frontend/src/routes/AppRoutes.js

```diff
@@ -3,12 +3,14 @@ import { Route, Switch } from 'react-router-dom';
 
 import NavBar from 'views/NavBar';
 import Home from 'views/Home';
+import RegisterUser from 'views/RegisterUser';
 
 const AppRoutes = () => (
   <>
     <NavBar />
     <Switch>
       <Route exact path='/movies' component={Home} />
+      <Route exact path='/movies/register' component={RegisterUser} />
```

Comment:
I think this route should be just
```suggestion
      <Route exact path='/register' component={RegisterUser} />
```

Reply:
Solved